File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Use nested for loops?
    int count = 0;
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1);
    while (count < 1)
    {
        if (0 <= n & n <= 8)
        {

            for (int i = 0 ; i < n; i++)
            {
                for (int y = 0; y < (n - i) - 1; y++)
                {
                    printf(" ");
                }
                for (int x = 0; x < i + 1; x++)
                {
                    printf("#");
                }
                printf("  ");
                for (int j = 0; j < i + 1; j++)
                {
                    printf("#");

                }
                printf("\n");
            }
            count += 1;
        }
    else {
        n = get_int("Height: ");
    }
    }
}

File: sentimental-mario-more/mario.py
count = 0
n = 0

while (n < 1) & (count < 1):
    try:
        height = int(input("Height: "))
    except:
        height = int(input("Height: "))
    if (0 < (height) <= 8):
        n += 1
        for i in range(height):
            for c in range((height - i) - 1):
                print(" ", end="")
            for v in range(i + 1):
                print("#", end="")
            print("  ", end="")
            for q in range(i + 1):
                print("#", end="")
            print("")
            n += 1
